lib/elevator.js

```diff
@@ -28,6 +28,7 @@
 
 var child_process = require('child_process');
 var commandBuilder = require('./command');
+var debug = require('debug')(require('../package.json').name);
```

Comment:
Haha yeah. We could hardcode the name, but it guess its a bit redundant if we can easily fetch it from package.json.